### PR TITLE
Scala 2.13

### PR DESCRIPTION
**Pull request by dlinov**

### CODE REVIEW

File: version.sbt [hunk 1]
```diff
@@ -1 +1 @@
-version in ThisBuild := "0.1.3-SNAPSHOT"
+version in ThisBuild := "0.1.3"
```
Comment (migesok) [inline version.sbt]: The version is set by the release plugin, better leave it as it is.

File: README.md [hunk 2]
```diff
@@ -2,22 +2,22 @@ Pre-canned
...
+cross-compiled for 2.12 and 2.13.
 
 Introduction
 ------------
 
 Pre-canned helps you to mock out the HTTP services your application depends on. This can be especially useful
 for your integration testing.
 
-For SBT add the dependency `"io.github.a-fistful-of-code" %% "pre-canned" % "0.1.1" % "test"`
+For SBT add the dependency `"io.github.a-fistful-of-code" %% "pre-canned" % "0.1.3" % "test"`
```
Comment (migesok) [inline README.md]: I will fix up the version when the release is done, please return this one as it was.

File: src/test/scala/com/netaporter/precanned/FancyDslSpec.scala [hunk 3]
```diff
@@ -40,7 +42,8 @@ class FancyDslSpec
   }
 
   "path expectation" should "match path" in {
-    animalApi expect path("/animals") and respond using resource("/responses/animals.json") end()
+    animalApi expect path("/animals") and respond using
+      resource("/responses/animals.json") end(defaultBlockUpTo)
```
Comment (migesok) [inline src/test/scala/com/netaporter/precanned/FancyDslSpec.scala]: Why did you need to add `defaultBlockUpTo`?

**Review by migesok — CHANGES_REQUESTED** [review]

File: version.sbt (hunk 1 above)
Comment (dlinov) [reply version.sbt]: Fixed as it was before

File: README.md (hunk 2 above)
Comment (dlinov) [reply README.md]: Changed back to 0.1.1

File: src/test/scala/com/netaporter/precanned/FancyDslSpec.scala (hunk 3 above)
Comment (dlinov) [reply src/test/scala/com/netaporter/precanned/FancyDslSpec.scala]: It fails to compile for me with scala 2.13 with empty braces:
```
[error] /pre-canned/src/test/scala/com/netaporter/precanned/FancyDslSpec.scala:139:84: type mismatch;
[error]  found   : Unit
[error]  required: scala.concurrent.duration.FiniteDuration
[error]     availablePortApi expect get and path("/status") and respond using entity("OK") end()
```
I couldn't come up with actual reason why it stopped compiling using scala 2.13 and used timeout explicitly

Comment (migesok) [reply src/test/scala/com/netaporter/precanned/FancyDslSpec.scala]: We should mention it the README `fancy` part, that `end()` works only for 2.12 and for 2.13 the example should be different.

Comment (dlinov) [reply src/test/scala/com/netaporter/precanned/FancyDslSpec.scala]: I found out you can't use call nullary methods with postfix notation: https://github.com/scala/bug/issues/11461,
so I decided to split `end` into two methods: one accepts custom timeout, the other one uses default timeout and can be called without empty braces. I know it's preferred to call methods with side effects using empty braces, but it's a trade-off to keep fancy dsl dotless.
